docs/core/reference/cored.md

```diff
@@ -0,0 +1,54 @@
+# cored Command
+
+Command `cored` provides the Chain Core daemon and API server.
+
+## Installation
+
+`cored` is included with all desktop installations of Chain Core
+Developer Edition, and is run by default when starting Mac, Windows and Docker.
+
+### From source
+
+_The instructions in this section require having the Go programming environment installed and the PATH variable correctly configured. See the [Chain Core Readme file](https://github.com/chain/chain/blob/main/Readme.md#building-from-source) for details._
+
+There are four build tags that change the behavior of the resulting binary:
+
+  - `reset`: allows the core database to be reset through the api
+  - `loopback_auth`: allows unauthenticated requests on the loopback device (localhost)
+  - `no_mockhsm`: disables the MockHSM provided for development
+  - `plain_http`: allows plain HTTP requests
+
+The default build process creates a binary with three build tags enabled for a
+friendlier experience. To build a binary from source with a set of build tags,
+use the following command:
+
+```sh
+go build -tags 'loopback_auth plain_http reset' chain/cmd/cored
+```
+
+## Flags
+
+* **-version**: Print version and build information
+
+## Environment Variables
+
+* **LISTEN_ADDR**: Address the Chain Core server will listen on, defaults to `:1999`
+* **DATABASE_URL**: URL of the Postgres database, defaults to `postgres:///core?sslmode=disable`
+* **CHAIN_CORE_HOME**: Path to Chain Core data directory on local file system, defaults to `$HOME/.chaincore`
+* **ROOT_CA_CERTS**
+* **SPLUNKADDR**
+* **LOGFILE**
+* **LOGSIZE**
+* **LOGCOUNT**
+* **LOG_QUERIES**
+* **MAXDBCONNS**
```

Comment:
this limits the number of simultaneous connections to postgres. it is a good idea to set it to something larger than the default in production, something like 100 or 200. the default is 10.

but we should reevaluate the default. we should also make new measurements about what the recommended setting would be for production.